src/templates/dataGridTableRow.jsx

```diff
@@ -27,30 +39,42 @@ const propTypes = {
 };
 
 const defaultProps = {
-    handle: undefined,
+    classes: null,
+    handle: false,
     isClickable: undefined,
     rowProps: undefined,
 };
 
-class DataGridTableRow extends React.PureComponent {
+const styles = {
+    handleGrid: {
+        flexWrap: 'nowrap !important',
+    },
+};
+
+class DataGridTableRow extends React.Component {
     constructor() {
         super();
 
         this.onClick = this.onClick.bind(this);
     }
 
     onClick() {
-        const { isClickable, row, rowProps } = this.props;
-        const { onClick } = rowProps;
+        const {
+            isClickable,
+            row,
+            rowProps: { onClick },
+        } = this.props;
+
         const isTextSelect = window.getSelection().toString();
```

Comment:
This probably works, but is a little bit gross, because technically at this point you have a string, not a Boolean.  I'd maybe like it slightly better as `!isEmpty(window.getSelection().toString())` or `!!window.getSelection().toString()`.

Probably a NAB since this is existing code, and not really being changed as part of this PR.

Reply:
Fixed.

Reply:
Looks good.  Resolving.